Fixed some typos and formatting. (#9)
danielehc/katacoda-scenarios: vault-consul-acl/finish.md

## Review

In this hands-on lab, you deployed a secure Consul datacenter using

> Vault to generate and manage certificates.

Reviewer: ```suggestion
Vault to generate and manage ACL tokens.
```

> The lab guided you through the steps necessary to deploy Consul
> with TLS encryption enabled to secure access to the UI, API, CLI,
> services, and agents.

Reviewer: ```suggestion
The lab guided you through the steps necessary to deploy Consul
with ACL enabled to verify identity of your server nodes and assign them necessary permissions.
```

Specifically, you:

> 

Reviewer: ```suggestion
- Started a Vault dev instance
- Started a Consul datacenter with ACL enabled
- Bootstrapped ACLs in Consul
- Created a Consul policy for servers
- Enabled Consul secrets engine in Vault
- Created a management token for Vault
- Created a Vault role to map Consul policy
- Created a Vault token associated with the role
- Verified the token was present in Consul and applied it to the agent
```

> - Started a Vault dev instance
> - Created a policy in Vault to allow certificate generation

Reviewer: ```suggestion
```

> - Enabled the PKI engine in Vault
> - Initialized the CA and generate an intermediate certificate
> - Generated certificates for your Consul servers
> - Used consul-template to retrieve certificates at runtime
> - Performed a certificate rotation

Reviewer: ```suggestion
```
